src/Mod/Mesh/Gui/RemoveComponents.cpp

```diff
@@ -23,13 +23,13 @@
 
 #include "PreCompiled.h"
 
-#ifndef _PreComp_
-# include <QPushButton>
+#ifndef _PreComp_
+# include <QPushButton>
 #endif
```

Comment:
Not sure why the contents of src/Mod/Mesh/Gui/RemoveComponents.cpp doesn't show the whitespace diff. Most of the correction on this file have been to whitespace with 1 typo fix interspersed.